include/nbl/asset/utils/IMeshManipulator.h

```diff
@@ -73,6 +73,8 @@ class IMeshManipulator : public virtual core::IReferenceCounted
 			float wage;												//angle wage of the vertex
 			core::vector4df_SIMD position;							//position of the vertex in 3D space
 			core::vector3df_SIMD parentTriangleFaceNormal;			//
+			core::vector3df_SIMD v0s;
+			core::vector3df_SIMD v0t;
```

Comment:
do not mix PR's together, I know this is for tangents.

Reply:
Heck sorry forgot to delete that bit

Reply:
resolved